.github/workflows/pylint-check.yml

```diff
@@ -0,0 +1,30 @@
+name: Pylint
+
+on:
+  pull_request:
+    branches: [master]
+    types: [opened, synchronize, edited]
+
+  push:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10"]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
```

Comment:
One more nit: can you lock this to a specific pylint version (at least a major version) so the check doesn't change when pylint updates?

Reply:
Sure, updated to `pip install pylint==3.1.0`